neutrino-custom/production.js

```diff
@@ -1,48 +1,26 @@
 'use strict';
-const webpack = require('webpack');
-const basePreset = require('./base');
-const CopyPlugin = require('copy-webpack-plugin');
-const CleanPlugin = require('clean-webpack-plugin');
-const CWD = require('./base').CWD;
-const UI = require('./base').UI;
-const DIST = require('./base').DIST;
-
-// The service domain is used to determine whether login is available in the auth component.
-let SERVICE_DOMAIN = process.env.SERVICE_DOMAIN;
+const { DefinePlugin } = require('webpack');
+const copy = require('neutrino-middleware-copy');
 
 module.exports = neutrino => {
-    basePreset(neutrino);
-
-    neutrino.config.plugin('minify')
-        .inject(BabiliPlugin => new BabiliPlugin({
-            evaluate: false, // prevents some minification errors
-        }
-    ));
-
-    // Define the service domain globally so that window.thServiceDomain can be set:
-    neutrino.config.plugin('define')
-        .use(webpack.DefinePlugin, {
-            SERVICE_DOMAIN: JSON.stringify(SERVICE_DOMAIN)
-        });
-
+    neutrino.config
+        .plugin('minify')
+        // prevents some minification errors
+            .tap(() => [{ evaluate: false }]).end()
```

Comment:
Without this, I got some errors in minified code that were just too cryptic for me to figure out - for example: 

```
TypeError: Cannot read property 'revision' of undefined
    at index.7905b7e….bundle.js:35
    at Al (vendor.103bf08….bundle.js:35)
    at vendor.103bf08….bundle.js:35
    at Rl.$eval (vendor.103bf08….bundle.js:35)
    at Rl.$digest (vendor.103bf08….bundle.js:35)
    at Rl.$apply (vendor.103bf08….bundle.js:35)
    at vendor.103bf08….bundle.js:35
    at Al (vendor.103bf08….bundle.js:35)
    at vendor.103bf08….bundle.js:35
```

After some guesssing and checking, I determined this was coming from the eval option (corresponds to this [package](https://github.com/babel/babili/tree/master/packages/babel-plugin-minify-constant-folding)). I'm thinking this may be easier to fix once sourcemaps are present, but if anyone has an idea what could be making this happen, I'm happy to make a PR with a fix!